tests/test_packageAlerts.py

```diff
@@ -26,9 +26,15 @@
 import shutil
 import tempfile
 import unittest
+from unittest.mock import patch, Mock
+import sys
 
 from astropy import wcs
 from astropy.nddata import CCDData
+try:
+    import confluent_kafka
+except ImportError as error:
+    error.msg += ("Could not import confluent_kafka. Alerts will not be sent to the alert stream")
```

Comment:
Again, this error message isn't being used.

Reply:
The import has been moved inside the `__init__` to take advantage of `self.logger`.

Reply:
The import now is back with the main imports and sets `confluent_kafka = None` when it fails. This then triggers an error message to be logged if `doProduceAlerts` was set.